packages/maker.js/src/core/dxf.ts

```diff
@@ -147,21 +147,20 @@ namespace MakerJs.exporter {
             return polylineEntity;
         }
 
-        function mtext(caption: ICaption) {
+        function text(caption: ICaption & { layer?: string }) {
             const center = point.middle(caption.anchor);
-            const mtextEntity: DxfParser.EntityMTEXT = {
-                type: "MTEXT",
-                position: {
-                    x: round(center[0], opts.accuracy),
-                    y: round(center[1], opts.accuracy)
-                },
-                height: opts.fontSize,
+            const textEntity: DxfParser.EntityTEXT = {
+                type: "TEXT",
+                startPoint: appendVertex(center, null),
+                endPoint: appendVertex(center, null),
+                layer: defaultLayer(caption.anchor, caption.layer),
```

Comment:
I think the caption.anchor behavior should be moved to model.getAllCaptionsOffset()